admin/pre_reg/views.py

```diff
@@ -122,12 +116,10 @@ def reject_draft(request, draft_pk):
 
     user = request.user.osf_user
     draft.reject(user)
-    return JsonResponse({})
-
+    return redirect('pre_reg:prereg', permanent=True, page_number=int(request.POST.get('page', 1)))
```

Comment:
Is there anyway something can be un-rejected? If so you might want a 302 not a 301.